src/Ontology/Oep4.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Oep4.h"
+#include <HexCoding.h>
+#include <cstdio>
+
+using namespace TW;
+
+namespace TW::Ontology {
+Oep4::Oep4(const Address addr) {
+    oep4Contract = Data(std::begin(addr._data), std::end(addr._data));
+}
+
+Transaction Oep4::romethod0(std::string method_name, uint32_t nonce) {
+    auto builder = ParamsBuilder();
+    Address addr(std::vector<uint8_t>(oep4Contract.begin(), oep4Contract.end()));
```

Comment:
you can just write `Address addr(ope4contract);`

Reply:
Will do